abis_mapping/templates/survey_metadata/mapping.py

```diff
@@ -273,18 +279,35 @@ def add_survey(
         graph.add((uri, rdflib.PROV.hadPlan, survey_method))
 
         # Add taxonomic coverage
-        if taxonomic_coverage := row["taxonomicCoverage"]:
-            graph.add((uri, utils.namespaces.BDR.target, rdflib.Literal(taxonomic_coverage)))
+        if taxonomic_coverage := row["targetTaxonomicScope"]:
+            for taxa in taxonomic_coverage:
+                graph.add((uri, utils.namespaces.BDR.target, rdflib.Literal(taxa)))
+
+        # Add targetHabitatScope
+        if habitats := row["targetHabitatScope"]:
+            for habitat in habitats:
+                graph.add((uri, utils.namespaces.BDR.basedAt, rdflib.Literal(habitat)))
 
         # Add purpose
-        if purpose := row["purpose"]:
+        if purpose := row["surveyPurpose"]:
             graph.add((uri, utils.namespaces.BDR.purpose, rdflib.Literal(purpose)))
 
         # Add keywords
         if keywords := row["keywords"]:
             for keyword in keywords:
                 graph.add((uri, rdflib.SDO.keywords, rdflib.Literal(keyword)))
 
+        # Add Orgs
+        if organisations := row["surveyOrganisation"]:
+            for organisation in organisations:
+                graph.add((uri, utils.namespaces.BDR.organization, rdflib.Literal(organisation)))
+
```

Comment:
@joecrowleygaia  what do you think this (above and below)?

Reply:
these are all new mappings? it's probably the way to go and i'd love to approve for completeness but there are the abis data standards that need to be adhered to which mieke is the main contact for us on such things. I'm happy with all the changes that happened yesterday so if you want you can revert this latest commit and i can merge. Then i'd recommend raising with Mieke at stand up tomorrow or via a backlog ticket at least to have these fields that haven't been mapped, to be.

Reply:
Yep, they were the new mappings that were not already covered by the previous branch.  I'll roll it back and save it for later :)